packages/account/src/Sections/Security/LoginHistory/login-history.tsx

```diff
@@ -0,0 +1,158 @@
+        </Table.Header>
+        <Table.Body>
+            {login_history.map(item => (
+                <Table.Row className='login-history__table__row' key={item.id}>
+                    <Table.Cell>
+                        <Text line_height='xs' size='xs'>
+                            {item.date}
+                        </Text>
+                    </Table.Cell>
+                    <Table.Cell className='login-history__table__row__cell login-history__table__row__cell--action'>
+                        {item.action}
+                    </Table.Cell>
+                    <Table.Cell>{item.browser}</Table.Cell>
+                    <Table.Cell>{item.ip}</Table.Cell>
+                    <Table.Cell>{item.status}</Table.Cell>
+                </Table.Row>
+            ))}
+        </Table.Body>
+    </Table>
+);
+
+const renderList = (fields: TGetFields, login_history: TData) => {
+    return (
+        <Table className='login-history__list'>
+            <Table.Body>
+                {login_history.map(item => (
+                    <div className={classNames('login-history__list__wrapper')} key={item.id}>
+                        <Table.Row className='login-history__list__row login-history__list__row--with-margin'>
+                            <Table.Cell className='login-history__list__row__cell'>
+                                <ListCell title={fields.date} text={item.date} />
+                            </Table.Cell>
+                        </Table.Row>
+                        <Table.Row className='login-history__list__row login-history__list__row--with-margin'>
+                            <Table.Cell className='login-history__list__row__cell'>
+                                <ListCell
+                                    className='login-history__list__row__cell--browser'
+                                    title={fields.browser}
+                                    text={item.browser}
+                                />
+                            </Table.Cell>
+                            <Table.Cell className='login-history__list__row__cell'>
+                                <ListCell
+                                    className='login-history__list__row__cell--action'
+                                    title={fields.action}
+                                    text={item.action}
+                                />
+                            </Table.Cell>
+                        </Table.Row>
+                        <Table.Row className='login-history__list__row'>
+                            <Table.Cell className='login-history__list__row__cell'>
+                                <ListCell
+                                    className='login-history__list__row__cell--ip-value'
+                                    title={fields.ip}
+                                    text={item.ip}
+                                />
+                            </Table.Cell>
+                            {isDesktop() && (
```

Comment:
i saw is_desktop in someones(probably @utkarsha-deriv) PR, added to the ui store. if you are aware, please try to take it from ui store.

Reply:
I didn't saw any is_desktop in our code base 🤔

Reply:
it haven't reached master branch yet. may be in the future we can fix it, if you are not using it now. 👌🏼

Reply:
sure 🙆‍♂️